app/app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
+    'Chitter - Speak Your Mind'
+  end
+
+  get '/peeps/new' do    
+    erb :'peeps/new'    
+  end
+
+  post '/peeps' do
+    @peep = Peep.create(content: params[:content])
+    redirect '/peeps'
+  end
+
+  get '/peeps' do     
+    @peeps = Peep.order_by_date 
+    erb :'peeps/index'
```

Comment:
it supposed to redirect to a page if there is any database stored?  right now its 404 maybe i dont have any table created?